services/mgmt/compute/examples/vm_list_using_resource_graph.rs

```diff
@@ -0,0 +1,85 @@
+use std::sync::Arc;
+use azure_identity::DefaultAzureCredential;
+use azure_mgmt_resourcegraph::models::{QueryRequest, QueryRequestOptions};
+use serde_json::{json, Value};
+use azure_mgmt_resourcegraph::Client as AzureResourceGraphClient;
+use azure_mgmt_resourcegraph::ClientBuilder as AzureResourceGraphClientBuilder;
```

Comment:
As this only uses a single `Client`, the alias isn't important here.  Additionally, using `Client::builder` below does not require importing `ClientBuilder` here.  If this was importing clients from multiple crates, differentiating the clients like this would be valuable.

```suggestion
use azure_mgmt_resourcegraph::Client;
```